etl-api/tests/common/test_app.rs

```diff
@@ -509,7 +510,11 @@ pub async fn spawn_test_app() -> TestApp {
 
     let key = generate_random_key::<32>().expect("failed to generate random key");
     let encryption_key = encryption::EncryptionKey { id: 0, key };
-    let api_key = "XOUbHmWbt9h7nWl15wWwyWQnctmFGNjpawMc3lT5CFs=".to_string();
+
+    // We choose a random API key from the ones configured to show that rotation works.
+    let api_key_index = random_range(0..config.api_keys.len());
+    let api_key = config.api_keys[api_key_index].clone();
```

Comment:
Decided to just inject one random key each time so that we can validate the rotation with no changes in tests.